book/src/testnet-participation.md

```diff
@@ -29,25 +29,33 @@ MacOS or WSL users may build from source.
 The shell commands in this section assume the following environment variables are
 set:
 ```bash
-$ export release=0.12.4
 $ export ip=$(dig +short beta.testnet.solana.com)
 $ export USE_INSTALL=1
 ```
 
 #### Obtaining The Software
-Prebuilt binaries are available for Linux x86_64 systems.  Download and install by running:
+##### Download Prebuilt Binaries
+Binaries are available for Linux x86_64 systems.  Download the binaries by navigating to:
+```
```

Comment:
Looks like CI is red because mdbook thinks that these blocks is Rust source code that needs to be compiled and checked.  Maybe drop the ``` for a slightly different format